TA_dataset/bin/dataset_search_command.py

```diff
@@ -436,6 +442,20 @@ def generate(self):
                         "DataSetFunction=makeRequest, destination={}, startTime={}"
                         .format(ds_url, time.time())
                     )
+                    if acct_dict.get(ds_acct).get("tenant") is not None:
```

Comment:
another code duplication

Reply:
same here

Reply:
This duplication is across multiple files